packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx

```diff
@@ -57,7 +57,7 @@ export const PageHeader = ({
         const navOpen = isManagedSidebar ? managedIsNavOpen : isNavOpen;
 
         return (
-          <header role="banner" className={css(styles.pageHeader, className)} {...props}>
+          <header className={css(styles.pageHeader, className)} {...props}>
```

Comment:
Can we allow this attribute to be omitted by default, but set if the user provides a value explicitly?

Reply:
Absolutely! Will open up a separate issue for adding a new prop that will do this.

Reply:
Ty!